app/src/main/java/com/example/notebook/view/fragments/my_notes/creation_note/CreationNoteFragment.kt

```diff
@@ -21,21 +23,16 @@ class AddingNoteFragment : BaseFragment<FragmentAddNoteBinding>(FragmentAddNoteB
 
             buttonSave.setOnClickListener {
                 saveNote()
-                findNavController().navigateUp()
             }
         }
     }
 
     private fun saveNote() {
         with(binding) {
-            if (isNoteNotEmpty()) {
-                val note = Note(
-                    DEFAULT_ID,
-                    editTitle.text.toString(),
-                    editNote.text.toString()
-                )
-
+            if (!isNoteEmpty()) {
+                val note = Note(editTitle.text.toString(), editNote.text.toString())
```

Comment:
yet again it seems odd to create a note on UI layer this way, this way you can introduce lot's storage related issues by manually handling creation of the class